src/Roslyn/Core/Microsoft.Net.RoslynDiagnostics.nuspec

```diff
@@ -0,0 +1,32 @@
+﻿<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.Net.RoslynDiagnostics</id>
+    <description>
+      Roslyn Diagnostics Toolset package. Contains analyzers specific to the .NET Compiler Platform ("Roslyn") project.
+
+      Supported Platforms:
+      - .NET Framework 4.5
+    </description>
+    <dependencies>
+      <dependency id="Microsoft.Net.ToolsetCompilers" version="$currentVersion$" />
```

Comment:
@tmeschter @jaredpar  Will these two package versions always be in sync or we need to manually assign a fixed version here?